test-tools/issuer-front-end/src/Main.tsx

```diff
@@ -1574,25 +1571,7 @@ export default function Main(props: WalletConnectionProps) {
                                             url: credentialMetaDataURL,
                                         };
 
-                                        const attributes: Attribute = {};
-
-                                        attributeSchema.forEach((obj) => {
-                                            if (obj.value !== undefined) {
-                                                if (obj.type === 'string') {
-                                                    attributes[obj.tag] = obj.value;
-                                                } else if (obj.type === 'number') {
-                                                    attributes[obj.tag] = BigInt(obj.value);
-                                                } else {
-                                                    setParsingError(
-                                                        `Attribute ${obj.tag} has type ${obj.type}. Only the types string/number are supported.`
-                                                    );
-                                                    throw new Error(
-                                                        `Attribute ${obj.tag} has type ${obj.type}. Only the types string/number are supported.`
-                                                    );
-                                                }
-                                            }
-                                        });
-
+                                        const attributes = parseAttributesFromForm(attributeSchema, setParsingError);
```

Comment:
The last test case could be reduced in code as well when using `parseAttributesFromForm()` and using `{renderAddPrompt(item)}:`